mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageManager.java

```diff
@@ -439,9 +470,10 @@ private int getBodyStartOctet(BodyOffsetInputStream bIn) {
         return bodyStartOctet;
     }
 
-    private AppendResult createAndDispatchMessage(Date internalDate, MailboxSession mailboxSession, File file, PropertyBuilder propertyBuilder, Flags flags, int bodyStartOctet) throws IOException, MailboxException {
-        try (SharedFileInputStream contentIn = new SharedFileInputStream(file)) {
-            final int size = (int) file.length();
+    private AppendResult createAndDispatchMessage(Date internalDate, MailboxSession mailboxSession, Content content, PropertyBuilder propertyBuilder, Flags flags, int bodyStartOctet) throws IOException, MailboxException {
+        // TODO MailboxMessage should be backed by a "content"
+        try (SharedFileInputStream contentIn = null) {
```

Comment:
this commit breaks that code on purpose? is it fixed later?

Reply:
Yes. And it get fixed later. (I assume it).